test/kcli/configure-ovs-network-test.bats

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bats
+
+
+#cleanUpCmd="echo 'empty cleanup'"
+
+setup() {
+
+    load 'bats/bats-support/load'
+    load 'bats/bats-assert/load'
+    load 'bats/bats-file/load'
+
+    DIR="$( cd "$( dirname "$BATS_TEST_FILENAME" )" >/dev/null 2>&1 && pwd )"
+   
+    cleanUpCmd=""
+}
+
+teardown() {
+    run ${cleanUpCmd}
+}
+
+@test "Single NIC" {
+    output_dir=${DIR}/_output/single-nic
+    rm -rf ${output_dir}
```

Comment:
always quote?  `"${output_dir}"` and maybe `--` guard for extra paranoia.  `rm -rf -- `

Reply:
Good point! fixing